isb_cgc/urls.py

```diff
@@ -19,16 +19,14 @@
 from django.urls import path
 from django.contrib import admin
 from django.conf import settings
-from django.views.generic.base import TemplateView
 
 from . import views
 
 admin.autodiscover()
 
 urlpatterns = [
 
-    url(r'^$', views.landing_page, name='landing_page'),
-    url(r'robots.txt', TemplateView.as_view(template_name="robots.txt", content_type="text/plain"), name='robots'),
```

Comment:
Did we add the robots.txt in elsewhere?

Reply:
robots.txt along with the url code are added back.